### ISSUE TITLE
new redirects to 404's distro abandoned?

### ISSUE THREAD

**Issue by twillis** [KXStudio/Website#10]
Hello, was trying to update kxstudio on my machine for the last couple of days and been getting

`
W: The repository 'http://kxstudio.linuxaudio.org/repo stable Release' does not have a Release file.
N: Data from such a repository can't be authenticated and is therefore potentially dangerous to use.
N: See apt-secure(8) manpage for repository creation and user configuration details.
E: Failed to fetch http://kxstudio.linuxaudio.org/repo/dists/stable/non-free/binary-amd64/Packages  404  Not Found
E: Some index files failed to download. They have been ignored, or old ones used instead.
`

which seems to be caused by this commit. 
https://github.com/KXStudio/Website/commit/afdfb4a2b6b28cea0bbdcc8133abbff946672686

and there's no mention of kxstudio on http://kxstudio.linuxaudio.org so i'm really confused. has this distro been abandoned?

**Comment by twillis** [comment]
just in case someone else arrives here by the same reason....

https://www.reddit.com/r/linuxaudio/comments/7umhjq/kxstudio/

**Comment by falkTX** [comment]
this must be a new low... I'm impressed.

a few days offline and we already considered abandoned. if you had search for in the forums, you'd know that linuxaudio.org was attacked and all its services are down at the moment.
the repositories still work fine, with 1 warning about the external repo being offline.

I guess I have to update the kxstudio-repos package to stop people from asking the same thing over and over again.
and I haven't checked facebook yet, I bet it's worse there.

TLDR: full linuxaudio.org is down, please be patient.

**Comment by twillis** [comment]
meant no disrespect, just trying to figure out why my software wasn't updating. as a newish kxstudio user, i was unaware of any forums because the software has worked flawlessly for the last year, i had no need to look until now. sorry you got hacked, sorry I added to your frustration by asking about abandonment(apparently an often touchy subject I was not aware of). was just doing my best to try to figure things out.

**Comment by falkTX** [comment]
It was not kxstudio website that got hacked, it was the entire linuxaudio.org servers.
This means libremusicproductions, the linuxaudio planet, its mailing lists, the emails and a few other things.

The kxstudio software is still updating fine right now, you only get a warning about a repository source being offline.
I've pushed an updated kxstudio-repos that will temporarily get rid of the external repo so the warning won't be shown next time you run through the updates.
(should appear as update in around 1 hour)

**Comment by twillis** [comment]
okkkkkkkkkkkkkk
